### PR TITLE
Rename phylorthology -> noveltree

### PR DESCRIPTION
**Pull request by mertcelebi**

### CODE REVIEW

File: noveltree/tsar_downsamp_test_parameters.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "input": "https://github.com/Arcadia-Science/tsar-phylorthology/raw/main/tsar-downsamp-test/tsar_downsamp_test_samplesheet.csv",
+  "input": "https://github.com/Arcadia-Science/2023-tsar-noveltree/raw/main/tsar-downsamp-test/tsar_downsamp_test_samplesheet.csv",
```
Comment (mertcelebi) [inline noveltree/tsar_downsamp_test_parameters.json]: @austinhpatton make sure this will work!

Comment (austinhpatton) [reply noveltree/tsar_downsamp_test_parameters.json]: I think this should actually be:
`"input": "https://github.com/Arcadia-Science/test-datasets/raw/main/noveltree/tsar_downsamp_test_samplesheet.csv"`

Right?

Comment (austinhpatton) [reply noveltree/tsar_downsamp_test_parameters.json]: I think I should have fixed this with the recent commit. Don't think we can actually test it until after it's merged though

Comment (austinhpatton) [reply noveltree/tsar_downsamp_test_parameters.json]: Currently running it locally having changed the main path to this branch and it seems to be working!

**Review by austinhpatton — APPROVED** [review]
Okay, I believe everything should be updated to the new paths with the rename, and everything _should_ work! 

Should be good to merge.

**Pull request merged**